src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -40,6 +50,21 @@ public function dump(MessageCatalogue $messages, $options = array())
     }
 
     /**
+     * Set XLIFF tool element metadata.
+     *
+     * @link http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#tool_elem
+     *
+     * @param array attributes supported by the <tool> element
+     */
+    protected function setToolMetadata( array $meta ){
```

Comment:
@timwhitlock making things protected is generally the worse extension point ever (it hurts maintenance a lot because we need to maintain BC on them). It is better to design proper extension point where needed than allowing to hack into the internal logic by exposing it. Inheritance is most of the time not the proper extension point.